databricks/sdk/service/compute.py

```diff
@@ -2106,16 +2115,70 @@ def from_dict(cls, d: Dict[str, any]) -> 'GlobalInitScriptUpdateRequest':
                    script_id=d.get('script_id', None))
 
 
+@dataclass
+class InitScriptEventDetails:
+    cluster: Optional['List[InitScriptInfoAndExecutionDetails]'] = None
+    global_: Optional['List[InitScriptInfoAndExecutionDetails]'] = None
```

Comment:
Here is the corrected rename of the `global` field to `global_`

Reply:
Nice